backend/dataall/modules/datapipelines/cdk/datapipelines_pipeline.py

```diff
@@ -159,7 +158,7 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
         code_dir_path = os.path.realpath(
             os.path.abspath(
                 os.path.join(
-                    __file__, "..", "..", "..", "..", "blueprints", "data_pipeline_blueprint"
+                    __file__, "..", "..", "..", "..", "..", "blueprints", "data_pipeline_blueprint"
```

Comment:
I wonder if we should move `data_pipeline_blueprint` into the `datapipeline` module?

Reply:
I moved it. Besides here, its also referenced for copying in the Dockerfile, which I also updated